tests/conftest.py

```diff
@@ -103,3 +105,59 @@ def fixture(tmp_dir_session):
     globals()[f'package_{normalized}'] = generate_package_fixture(package)
     globals()[f'sdist_{normalized}'] = generate_sdist_fixture(package)
     globals()[f'wheel_{normalized}'] = generate_wheel_fixture(package)
+
+
+@pytest.fixture(scope='session')
+def pep518_wheelhouse(tmpdir_factory):
+    wheelhouse = tmpdir_factory.mktemp('wheelhouse')
+    dist = tmpdir_factory.mktemp('dist')
+    subprocess.run(
+        [sys.executable, '-m', 'build', '--wheel', '--outdir', str(dist)],
+        cwd=str(package_dir.parent.parent),
+        check=True,
+    )
+    (wheel_path,) = dist.visit('*.whl')
+    subprocess.run(
+        [
+            sys.executable,
+            '-m',
+            'pip',
+            'download',
+            '-q',
+            '-d',
+            str(wheelhouse),
+            str(wheel_path),
+        ],
+        check=True,
+    )
+    subprocess.run(
+        [
+            sys.executable,
+            '-m',
+            'pip',
+            'download',
+            '-q',
+            '-d',
+            str(wheelhouse),
+            'build',
+            'tomli',
+            'wheel',
+            'colorama',
+            'meson',
+            'ninja',
+            'ninja',
+            'patchelf',
+            'pyproject-metadata',
+            'tomli',
+            'typing-extensions',
```

Comment:
tomli and ninja are repeated twice, dunno if that is intentional.

Reply:
Probably not.

Reply:
Fixed & sorted.